app/subscriber/src/components/content-list/ContentListContext.tsx

```diff
@@ -32,14 +33,16 @@ export const ContentListProvider: React.FC<IContentListProviderProps> = ({ child
   const [activeFileReference, setActiveFileReference] = useState<IFileReferenceModel>();
 
   React.useEffect(() => {
-    getAllContentInMyReports()
-      .then((reportContent) => {
-        storeReportContent(reportContent);
-      })
-      .catch(() => {});
+    if (authenticated) {
```

Comment:
Fixed bug on login page.  It's a little disappointing that this code is even called on the initial visit to the app...